def/de/menu_win32.cson

```diff
@@ -32,270 +32,270 @@ Menu:
       "Stylesheet…":
         value: "Stylesheet..."
       "&Save":
-        value: "Save(&S)"
+        value: "&Speichern"
       "Save &As…":
-        value: "Save As(&A)..."
+        value: "Speichern &als..."
       "Save A&ll":
-        value: "Save All(&L)"
+        value: "&Alles Speichern"
       "&Close Tab":
-        value: "Close Tab(&C)"
+        value: "Tab schließen"
       "Close &Pane":
-        value: "Close Pane(&P)"
+        value: "Alle Tabs schließen"
       "Clos&e Window":
-        value: "Close Window(&E)"
+        value: "F&enster schließen"
       "E&xit":
         value: "Exit(&X)"
       "Close All Tabs":
-        value: "Close All Tabs"
+        value: "Alle Tabs schließen"
   "&Edit":
-    value: "Edit(&E)"
+    value: "B&earbeiten"
     submenu:
       "&Undo":
-        value: "Undo(&U)"
+        value: "Rückgängig"
       "&Redo":
-        value: "Redo(&R)"
-      "Cu&t":
-        value: "Cut(&T)"
-      "&Copy":
-        value: "Copy(&C)"
+        value: "Wiederhe&rstellen"
+      "&Cut":

```

Comment:
Ah, I see. this was a copy / paste Error from the menu_linux.cson. Will fix this today later on.